shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
@shawndwells Our CentOS and RHEL setups are identical in terms of compliance. It allows for easy testing and migration but we are VERY clear that, if it's not certified, it's not valid protection.

Therefore, I think that there should be a separate check for FIPS (I think there is). No reason to fail this since we want to encourage best practice.

To be overly specific: https://github.com/simp/pupmod-simp-ssh/blob/master/manifests/server/params.pp